### DISCUSSION TITLE
Editing Sequences

### DISCUSSION THREAD

**Discussion by Lifeisgood7272** [onlaj/Piano-LED-Visualizer#263] [Q&A]
Hey,

So for editing sequences, I have to take the SD card out of the PI and edit it's files on a computer right? Is it safe to just take the SD card out?

Thanks

**Comment by onlaj** [comment]
Hello.
You can also connect to your RPi through SSH or FTP.
In near future it will be possible to edit sequences using web interface.

**Reply by Lifeisgood7272** [reply]
Thanks! And editing sequences with the web interface would be fricking dope, so keep on working on that definitely!

**Reply by Lifeisgood7272** [reply]
> Hello. You can also connect to your RPi through SSH or FTP. In near future it will be possible to edit sequences using web interface.

Hey,

I got the sequences working. At first it crashed whenever I got to a step that had a light speed of "Very Fast", I figured it was because I spelled it wrong, so I tried "Very_Fast" and it still crashed. I switched it to "Instant" and now it works. So my first question is, how do I spell "Very Fast" light speed on the sequences.xml file so the Pi can read it? I'll try "VeryFast" next and see if that counts. Same for Very Slow.

**Also, the program appears to be skipping every other sequence, even on the examples you gave.** This is my biggest problem. Is this a known bug or is something wrong with the way it's written? Right now I can work around it by having dummy sequences every other sequence so it plays only the ones I want, but right now I have a sequence of 29 steps so that will be 58 steps because of this issue! I attached the file as a .txt file, let me know if anything looks wrong. (.xml not supported on github)
[sequences - Copy.txt](https://github.com/onlaj/Piano-LED-Visualizer/files/7450334/sequences.-.Copy.txt)
.

**Comment by onlaj** [comment]
You can now edit sequences directly on web interface
https://github.com/onlaj/Piano-LED-Visualizer/pull/268
